src/components/TimeSeries/example-data/stations.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Move example data from the component's directory to `mocks`. You can create a `mocks` directory inside `src` and move mocks in there.